test/helpers.js

```diff
@@ -53,17 +47,14 @@ export function tryToReadFile(file) {
 	}
 }
 
+const { window } = new JSDOM('<main></main>');
+global.document = window.document;
```

Comment:
It's not new with this pull request, but I figured I would bring to your attention that the [jsdom README](https://github.com/tmpvar/jsdom#executing-scripts) (towards the bottom of that section) advises against putting properties from the DOM on the global scope. This might be the cause of the issue in the two tests where event listeners are executed on `global` rather than `window`.